README.md

```diff
@@ -20,6 +22,34 @@ Note that `node-gyp` must be installed. See the
 [node-gyp](https://github.com/nodejs/node-gyp) documentation for more
 information.
 
+### npm
+
+It is not recommended to install `hsd` from npm's repositories
+but it is still possible to install using `npm` using a remote
+`git` repository. To install a particular version, a git tag can
+be used. It is recommended to use the latest version.
+
+```
+$ npm install -g https://github.com/handshake-org/hsd.git#v2.1.3
+```
+
+If adding `hsd` as a dependency to a project, use the command:
+
+```
+$ npm install https://github.com/handshake-org/hsd.git#v2.1.3
```

Comment:
include `--save` ?

Reply:
@pinheadmz it depends on if we want to assume a reader understands how to use `npm`.

Reply:
`--save` is no longer required to add the the dependency to the `package.json`, I believe that is old `npm` behavior. You must pass `--no-save` to not add it.

```bash
$ npm --version
6.14.3

$ npm install --help

npm install (with no args, in package dir)
npm install [<@scope>/]<pkg>
npm install [<@scope>/]<pkg>@<tag>
npm install [<@scope>/]<pkg>@<version>
npm install [<@scope>/]<pkg>@<version range>
npm install <alias>@npm:<name>
npm install <folder>
npm install <tarball file>
npm install <tarball url>
npm install <git:// url>
npm install <github username>/<github project>

aliases: i, isntall, add
common options: [--save-prod|--save-dev|--save-optional] [--save-exact] [--no-save]
```